bc/assets/templates/includes/header.html

```diff
@@ -0,0 +1,51 @@
+ {% load static %}
+
+<div class="relative bg-gray-200 border-b-2 border-gray-400">
+    <div class="flex justify-between mx-auto px-4 sm:px-6">
+        <div class="flex w-full justify-between items-center py-6 md:space-x-6 lg:space-x-10">
+            <div class="absolute top-6">
+                <img src="{% static "svg/icon.svg" %}"  layout="responsive" class="rounded-lg border-4 border-black h-20 lg:h-24 bg-color-white"/>
```

Comment:
More black here. Let's get rid of all the black, please.

Reply:
Sure!